modules/node_modules/@ncigdc/containers/CaseTr.js

```diff
@@ -35,7 +36,17 @@ export type TProps = {|
   theme: Object,
 |};
 
-export const CaseTrComponent = ({ node, index, theme }: TProps) => {
+function massageRelayFiles(files: Array): Array {
+  return _.get(files, 'hits.edges', [])
```

Comment:
lol @ the function name. lodash-less `(files.hits || { edges : [] }).edges.map` instead? Oh and `files` is typed Array in the signature, should it be an Object?